### DISCUSSION TITLE
[Issue Forms] "Leave a comment" overlaps textarea placeholder

### DISCUSSION THREAD

**Discussion by TWiStErRob** [community/community#76763] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

The "Leave a comment" placeholder should only show up on issue/discussion threads, not in forms.

<img width="410" alt="image" src="https://github.com/community/community/assets/2906988/98d8c514-3b22-49f7-b884-c552fd33a245">

<img width="799" alt="image" src="https://github.com/community/community/assets/2906988/3efa4e9f-aca6-4f0f-ab4b-70ac9e145402">

It happens even in this discussion board:
<img width="902" alt="image" src="https://github.com/community/community/assets/2906988/9ab4edd8-93d0-44ef-b0e6-170c8069db35">

### Repro

1. https://github.com/orgs/community/discussions/new?category=projects-and-issues
2. Click in Body > Write > textarea

*Expected*: Able to read "STOP!"
*Actual*: "Leave a comment" overlays the textarea.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by awanlin** [comment]
Just wanted to share that we have a workaround for this, it's not ideal so having this fixed would be wonderful, but by using an `\n` we can move the `placeholder` text to a new line under this forced "Leave a comment" text

Here's an example of this being used: 

https://github.com/backstage/backstage/blob/ddbe7523890513574c00f5ccdc886784b6c15667/.github/ISSUE_TEMPLATE/bug.yaml?plain=1#L38-L41

**Reply by TWiStErRob** [reply]
Lol, what a hackaround! Kudos.

<details><summary>Tip for yaml</summary>

This should yield the same results, but with less escaping and shorter lines (see https://yaml-multiline.info/ for more):

```yaml
    attributes:
      label: '👟 Reproduction steps'
      description: >
        How do you trigger this bug?
        Please walk us through it step by step.
        Provide a link to a live example, or an unambiguous set of steps to reproduce this bug.
        Include code or configuration to reproduce, if relevant.
      placeholder: |
        
        1. Go to '...'
        2. Click on '....'
        3. Scroll down to '....'
```

(The line above 1. is indented to the same level.)

</details>

**Reply by awanlin** [reply]
TIL - Thanks for this YAML tip!

**Comment by TWiStErRob** [comment]
This looks to be fixed now.
